code/game/objects/items/devices/scanners/health.dm

```diff
@@ -117,13 +117,13 @@
 		if(H.status_flags & FAKEDEATH)
 			pulse_result = 0
 		else
-			pulse_result = H.get_pulse(GETPULSE_TOOL)
+			pulse_result = H.get_pulse_as_string(GETPULSE_TOOL)
```

Comment:
See line 121; `"extremely weak and fast, patient's artery feels like a threadbpm"` is invalid, `pulse_result` should maybe use `get_pulse_as_number()` instead.
```suggestion
			pulse_result = H.get_pulse_as_number()
```
Or modify/remove line 121 instead, since it *should* always be overwritten anyway.

Reply:
Nevermind, `GETPULSE_TOOL` is the more accurate one (terrible name) so it'll output `>250bpm` which is valid. I still don't like that but it's not enough to block over.